governance/internationals.md

```diff
@@ -0,0 +1,55 @@
+# International Bible Quizzing
+These rules apply to the Internationals quiz meet conducted once yearly, administered under the oversight of the CQLT.
+
+## Eligibility Requirements
+- All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.
+- In order to quiz at Internationals for a specific district, the quizzer must reside in that district or in a bordering community.  Their home church must be from the district in which they quiz.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+
+## Team Structure
+- Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send.  These will be the district International Teams.
+- A team will be composed of a maximum of five members: four starting quizzers and one substitute.  These five shall constitute the team for International Finals.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+- Each team must have a coach and may also have an assistant coach. 
+
+## Event Registration
+- All arrangements, promotion, and administration will be by the Life Impact Office.
+- Notification by each district of its intention to enter a team must be received by the Life Impact Office, on a form supplied by that office, no later than the date posted on the C&MA Quizzing website.
+- The names of the International Team, coaching staff, and all registration fees must be sent, when specified, to the Life Impact Office for that quiz year.
+
+## Selection of Officials
+- Officials for a sanctioned quiz may include a quizmaster, answer judge(s), and a scorekeeper(s).
+- The international quizmaster and officials will be appointed by the CQLT. To be considered for the CQLT appointment, all quizmasters and officials must:
+    - Be a regular church attender. 
+    - Have three years of district quizmastering experience and be recommended by their district office or quiz committee.
+    - Go through an evaluation process established by the CQLT, to determine that each official is qualified to perform their specific role.
+- When deemed necessary, the quizmaster or any official may be replaced from his/her position during a meet by the C&MA Quizzing Leadership Team. 
+- Team coaches whose teams are not involved in the quiz shall serve as answer judges and scorekeepers if needed.
+
+### Answer Judge
+- The head answer judge is responsible for the correctness of questions.
```

Comment:
Does this ever actually happen at IBQ?

Reply:
I kinda want to delete this. For a few reasons: (1) "head answer judge" is not a term that exists, AFAIK, (2) correctness of questions should be the responsibility of whoever is in charge of the question set for Internationals.

To your question Gryphon, I believe John Foley has in the past gone through all printed questions and checked them. (he doesn't catch everything invalid though).

Reply:
I'd want to remove it, and if anybody's so inclined to check validity, w00t. But seems like the CQLT should (and maybe does?) appoint a committee for this. Zach, what do you think?

Reply:
Yes. Technically Jon Foley has been formally asked a few times as the Room 1 Answer Judge to review questions. (He also enjoys looking them over) but we do appoint someone to handle the quiz generation and to check them for validity.  Heather Hobbie will most likely do that this year.

Reply:
I would remove it, as the position doesn't ever get created nor designated to an individual. All AJ & QMs are asked to review their packets before quizzing starts